<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>SaveMobame</class>
 <widget class="QMainWindow" name="SaveMobame">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>382</width>
    <height>308</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>382</width>
    <height>308</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>382</width>
    <height>308</height>
   </size>
  </property>
  <property name="font">
   <font>
    <family>Calibri</family>
   </font>
  </property>
  <property name="acceptDrops">
   <bool>true</bool>
  </property>
  <property name="windowTitle">
   <string>SaveMobame</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="saveBt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>319</x>
      <y>251</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Save</string>
    </property>
   </widget>
   <widget class="QPushButton" name="dirBt">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>2</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Dir</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="mailtextEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>25</y>
      <width>381</width>
      <height>201</height>
     </rect>
    </property>
    <property name="acceptDrops">
     <bool>false</bool>
    </property>
    <property name="placeholderText">
     <string>MailText</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="saveEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>3</y>
      <width>338</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Save Dir | Unix Format</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitBt">
    <property name="geometry">
     <rect>
      <x>255</x>
      <y>251</y>
      <width>61</width>
      <height>23</height>
     </rect>
    </property>
    <property name="text">
     <string>Exit</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="autosavecheckBox">
    <property name="geometry">
     <rect>
      <x>80</x>
      <y>255</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>AutoSave</string>
    </property>
    <property name="checked">
     <bool>false</bool>
    </property>
   </widget>
   <widget class="QCheckBox" name="lockedcheckBox">
    <property name="geometry">
     <rect>
      <x>5</x>
      <y>255</y>
      <width>81</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Locked</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="wordEdit">
    <property name="geometry">
     <rect>
      <x>1</x>
      <y>230</y>
      <width>380</width>
      <height>20</height>
     </rect>
    </property>
    <property name="placeholderText">
     <string>Word</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>382</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="font">
     <font>
      <family>Calibri</family>
     </font>
    </property>
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionSave"/>
    <addaction name="actionImage"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <addaction name="menuFile"/>
  </widget>
  <action name="actionSave">
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionImage">
   <property name="text">
    <string>Image</string>
   </property>
   <property name="font">
    <font>
     <family>Calibri</family>
    </font>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
